opmd_viewer/openpmd_timeseries/plotter.py

```diff
@@ -195,6 +204,10 @@ def show_field_2d(self, F, info, slicing_dir, m,
 
         geometry: string
            Either "2dcartesian", "3dcartesian" or "thetaMode"
+
+        plot_range : list of lists
```

Comment:
a list of a `.min` & `.max` dict (tuple) would probably be easier to read but ok :)